qa/workunits/ceph-helpers.sh

```diff
@@ -206,6 +217,8 @@ function test_kill_daemon() {
         kill_daemon $pidfile TERM || return 1
     done
 
+    # Wait a bit to get things reported down.
+    sleep 3
```

Comment:
@dachary 
I've seen this happen on very loaded hosts. Where there seemed to be a race and ceph-mon had not processed all status updates. Resulting in weird responses here.
But I'll remove it...